### ISSUE TITLE
Plugin tab

### ISSUE THREAD

**Issue by fernandogobah** [Ylianst/MeshCentral#3872]
The plugin tab is visible for all users. Is possible restricted for only administrators?

**Comment by Ylianst** [comment]
I don't support plug-in's. You can ask Denis who writes the plug-in's for support for that.

**Comment by prononext** [comment]
Yes, but just the possibility to have the plugins functionality visible or not seemes like a core permission function which is needed in MeshCentral. Inside Plugin permissions are of course part of the plugin developer.

**Comment by fernandogobah** [comment]
@si458 You can help it?

**Comment by si458** [comment]
@fernandogobah sadly meshcentral doesnt fully support plug-ins, and extending on plug-ins WONT happen.
sorry
